cli/lib/exec/spawn.js

```diff
@@ -81,7 +81,7 @@ module.exports = {
       args = [args]
     }
 
-    args = [...args, '--cwd', process.cwd()]
+    args = [...args, '--cwd', process.cwd(), '--cliNodePath', process.execPath, '--cliNodeVersion', process.versions.node]
```

Comment:
I would call these args `--userNodePath` and `--userNodeVersion` instead of `CLI`, since CLI is an implementation detail, but `user` is the actual descriptor.